packages/manager-react-components/src/components/redirection-guard/redirection-guard.component.tsx

```diff
@@ -8,23 +8,31 @@ export type RedirectionGuardProps = {
   condition: boolean;
   isLoading: boolean;
   route: string;
+  isError?: boolean;
+  errorComponent?: React.ReactNode;
 };
 
 export function RedirectionGuard({
   route,
   condition,
   isLoading,
   children,
+  isError,
+  errorComponent,
 }: RedirectionGuardProps): JSX.Element {
   if (isLoading) {
     return (
       <OsdsSpinner
         data-testid="redirectionGuard_spinner"
         size={ODS_SPINNER_SIZE.md}
-        inline={true}
+        inline
       />
     );
   }
 
+  if (isError && errorComponent) {
+    return <>{errorComponent}</>;
+  }
+
```

Comment:
why would you handle the error and display a component here ? i am not sure it's belong here

wouldn't it be better to handle that in the page that is importing the RedirectionGuard component ?